### PR TITLE
Update grids to V2 of config_grids xml format

### LINKED ISSUE #1833: Using invalid compset can cause confusion about exact radius of the earth in MPAS-CICE
Using a master as of yesterday. Can try with more recent if this is known?
This is on cori-knl.  Can also try edison to see if this is machine-specific? 
I'm pretty sure I (and others) have run this or something like this not that long ago, so it might be a recent change.

`./create_newcase -case /global/cscratch1/sd/ndk/acme_scratch/cori-knl/A_WCYCL1850S_ne30_oEC60to30v3_cori-knl01 -compset A_WCYCL1850S -res ne30np4_oEC60to30v3 -mach cori-knl`

Fails quickly, so can use debug Q.

```
cori05% cat run/log.cice.0000.err
----------------------------------------------------------------------
Beginning MPAS-cice Error Log File for task       0 of    5120
    Opened at 2017/10/11 14:54:58
----------------------------------------------------------------------

CRITICAL ERROR: rescale_mesh: inconsistent radii: sphere_radius: 6371230.00000000, earthRadius: 6371229.00000000
Logging complete.  Closing file at 2017/10/11 14:54:58
```

casedir:
`/global/cscratch1/sd/ndk/acme_scratch/cori-knl/A_WCYCL1850S_ne30_oEC60to30v3_cori-knl01`

### PR DESCRIPTION
**Pull request by jgfouca**
Update grids to V2

Fixes #1833 

[BFB]

### CODE REVIEW

**Comment by jgfouca** [comment]
@rljacob , please add more reviewers if you think it's appropriate.

File: cime/config/e3sm/config_grids.xml [hunk 1]
```diff
@@ -1,1262 +1,1611 @@
...
+    <model_grid alias="T31_g37">
+      <grid name="atm">T31</grid>
+      <grid name="lnd">T31</grid>
+      <grid name="ocnice">gx3v7</grid>
+      <grid name="rof">r05</grid>
+      <grid name="glc">null</grid>
+      <grid name="wav">null</grid>
+      <mask>gx3v7</mask>
+    </model_grid>
+
```
Comment (jonbob) [inline cime/config/e3sm/config_grids.xml]: this is a POP-CICE ocnice grid and not supported by mpas components

**Comment by rljacob** [comment]
version 2 of the config_grids xml format.  Not E3SM v2.

File: cime/config/e3sm/config_grids.xml [hunk 2]
```diff
@@ -1,1262 +1,1611 @@
...
+      <grid name="rof">r05</grid>
+      <grid name="glc">null</grid>
+      <grid name="wav">null</grid>
+      <mask>gx1v6</mask>
+    </model_grid>
+
+    <model_grid alias="ne30_oEC_ICG" compset="(CAM5.+CLM.+MPASO%SPUNUP)">
+      <grid name="atm">ne30np4</grid>
+      <grid name="lnd">ne30np4</grid>
+      <grid name="ocnice">oEC60to30ICG</grid>
```
Comment (jonbob) [inline cime/config/e3sm/config_grids.xml]: The ocnice grid name has been oEC60to30_ICG - I think leaving the underscore off will cause this to fail unless we make other changes to match

Comment (jgfouca) [reply cime/config/e3sm/config_grids.xml]: Nice catch, I think there may have been a bug in my converter script.

**Comment by jgfouca** [comment]
One major difference I'm noticing is that V2 is smart enough to realize that all stub components should have a null grid, whereas V1 was not. This is causing lots of nml diffs, but I don't think actual behavior of the model will be different.

**Comment by jgfouca** [comment]
@jonbob , I am a bit concerned about this nml diff I'm seeing for ERP_Ld3.ne30_oECv3_ICG.A_WCYCL1850:

```
2018-05-08 10:34:10: NLCOMP
Comparison failed between '/home/jgfouca/acme/scratch/ERP_Ld3.ne30_oECv3_ICG.A_WCYCL1850S.melvin_gnu.C.20180508_103201_ndr3zc/CaseDocs/lnd_in' with '/sems-data-store/ACME/baselines/gnu/master/ERP_Ld3.ne30_oECv3_ICG.A_WCYCL1850S.melvin_gnu/CaseDocs/lnd_in'
  BASE: fatmlndfrc = domain.lnd.ne30np4_oEC60to30v3.161222.nc'
  COMP: fatmlndfrc = domain.lnd.ne30np4_oEC60to30v3wLI_mask.170802.nc'
```

**Comment by jgfouca** [comment]
@jonbob , SMS.f09_g16_a.MALI.melvin_gnu.C is also significantly different:

```
Comparison failed between '/home/jgfouca/acme/scratch/SMS.f09_g16_a.MALI.melvin_gnu.C.20180508_103201_ndr3zc/CaseDocs/drv_in' with '/sems-data-store/ACME/baselines/gnu/master/SMS.f09_g16_a.MALI.melvin_gnu/CaseDocs/drv_in'
Differences in namelist 'cime_pes':
  BASE: atm_ntasks = 48
  COMP: atm_ntasks = 8
  BASE: cpl_ntasks = 48
  COMP: cpl_ntasks = 8
  BASE: glc_ntasks = 48
  COMP: glc_ntasks = 8
  BASE: ice_ntasks = 48
  COMP: ice_ntasks = 8
  BASE: lnd_ntasks = 48
  COMP: lnd_ntasks = 8
  BASE: ocn_ntasks = 48
  COMP: ocn_ntasks = 8
  BASE: rof_ntasks = 48
  COMP: rof_ntasks = 8
  BASE: wav_ntasks = 48
  COMP: wav_ntasks = 8
Differences in namelist 'seq_infodata_inparm':
  BASE: atm_gnam = '0.9x1.25'
  COMP: atm_gnam = 'null'
  BASE: ice_gnam = 'gx1v6'
  COMP: ice_gnam = 'null'
  BASE: lnd_gnam = '0.9x1.25'
  COMP: lnd_gnam = 'null'
  BASE: ocn_gnam = 'gx1v6'
  COMP: ocn_gnam = 'null'
  BASE: rof_gnam = 'r05'
  COMP: rof_gnam = 'null'
```

**Comment by jonbob** [comment]
@jgfouca - I'll dig in...

**Comment by jonbob** [comment]
@jgfouca - I'm really confused. Despite everything I tried to change in config_grids, the create_test for ERP_Ld3.ne30_oECv3_ICG.A_WCYCL1850S insists on using the oEC60to30v3wLI mask. Do you know how it's matching compsets and resolutions?

**Comment by jgfouca** [comment]
@jonbob , I'm talking to the cesg people and there's a problem with the mask field matching.

**Comment by jgfouca** [comment]
@jonbob , OK, I've fixes two crucial issues. The missing underscores have been fixes and the mask matching in grids.py now does exact matches. With these changes, all the cases in e3sm_integration look good except:

```
SMS.f09_g16_a.MALI.melvin_gnu.
SMS_D_Ld1.ne30_oECv3_ICG.A_WCYCL1850S_CMIP6.melvin_gnu.allactive-v1cmip6
SMS_R_Ld5.T42_T42.FSCM5A97.melvin_gnu
ERS.f09_g16_g.MALISIA.melvin_gnu
```

**Comment by jonbob** [comment]
Thanks @jgfouca - I'll go look at those few...

**Comment by jgfouca** [comment]
For SMS.f09_g16_a.MALI.melvin_gnu:

```
2018-05-08 12:44:36: NLCOMP
Comparison failed between '/home/jgfouca/acme/scratch/SMS.f09_g16_a.MALI.melvin_gnu.C.20180508_124325_4emhrx/CaseDocs/drv_in' with '/sems-data-store/ACME/baselines/gnu/master/SMS.f09_g16_a.MALI.melvin_gnu/CaseDocs/drv_in'
Differences in namelist 'cime_pes':
  BASE: atm_ntasks = 48
  COMP: atm_ntasks = 8
  BASE: cpl_ntasks = 48
  COMP: cpl_ntasks = 8
  BASE: glc_ntasks = 48
  COMP: glc_ntasks = 8
  BASE: ice_ntasks = 48
  COMP: ice_ntasks = 8
  BASE: lnd_ntasks = 48
  COMP: lnd_ntasks = 8
  BASE: ocn_ntasks = 48
  COMP: ocn_ntasks = 8
  BASE: rof_ntasks = 48
  COMP: rof_ntasks = 8
  BASE: wav_ntasks = 48
  COMP: wav_ntasks = 8
Differences in namelist 'seq_infodata_inparm':
  BASE: atm_gnam = '0.9x1.25'
  COMP: atm_gnam = 'null'
  BASE: ice_gnam = 'gx1v6'
  COMP: ice_gnam = 'null'
  BASE: lnd_gnam = '0.9x1.25'
  COMP: lnd_gnam = 'null'
  BASE: ocn_gnam = 'gx1v6'
  COMP: ocn_gnam = 'null'
  BASE: rof_gnam = 'r05'
  COMP: rof_gnam = 'null'
Comparison failed between '/home/jgfouca/acme/scratch/SMS.f09_g16_a.MALI.melvin_gnu.C.20180508_124325_4emhrx/CaseDocs/seq_maps.rc' with '/sems-data-store/ACME/baselines/gnu/master/SMS.f09_g16_a.MALI.melvin_gnu/CaseDocs/seq_maps.rc'
Differences in namelist 'seq_maps.rc':
  BASE: atm2ice_fmapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/fv0.9x1.25/map_fv0.9x1.25_TO_gx1v6_aave.130322.nc'
  COMP: atm2ice_fmapname = 'idmap'
  BASE: atm2ice_smapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/fv0.9x1.25/map_fv0.9x1.25_TO_gx1v6_blin.130322.nc'
  COMP: atm2ice_smapname = 'idmap'
  BASE: atm2ice_vmapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/fv0.9x1.25/map_fv0.9x1.25_TO_gx1v6_patc.130322.nc'
  COMP: atm2ice_vmapname = 'idmap'
  BASE: atm2ocn_fmapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/fv0.9x1.25/map_fv0.9x1.25_TO_gx1v6_aave.130322.nc'
  COMP: atm2ocn_fmapname = 'idmap'
  BASE: atm2ocn_smapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/fv0.9x1.25/map_fv0.9x1.25_TO_gx1v6_blin.130322.nc'
  COMP: atm2ocn_smapname = 'idmap'
  BASE: atm2ocn_vmapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/fv0.9x1.25/map_fv0.9x1.25_TO_gx1v6_patc.130322.nc'
  COMP: atm2ocn_vmapname = 'idmap'
  BASE: glc2lnd_fmapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/mpas.ais20km/map_ais20km_TO_f09_aave.151005.nc'
  COMP: glc2lnd_fmapname = 'idmap'
  BASE: glc2lnd_smapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/mpas.ais20km/map_ais20km_TO_f09_aave.151005.nc'
  COMP: glc2lnd_smapname = 'idmap'
  BASE: ice2atm_fmapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/gx1v6/map_gx1v6_TO_fv0.9x1.25_aave.130322.nc'
  COMP: ice2atm_fmapname = 'idmap'
  BASE: ice2atm_smapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/gx1v6/map_gx1v6_TO_fv0.9x1.25_aave.130322.nc'
  COMP: ice2atm_smapname = 'idmap'
  BASE: lnd2glc_fmapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/fv0.9x1.25/map_f09_TO_ais20km_aave.151005.nc'
  COMP: lnd2glc_fmapname = 'idmap'
  BASE: lnd2glc_smapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/fv0.9x1.25/map_f09_TO_ais20km_bilin.151005.nc'
  COMP: lnd2glc_smapname = 'idmap'
  BASE: lnd2rof_fmapname = '/sems-data-store/ACME/inputdata/lnd/clm2/mappingdata/maps/0.9x1.25/map_0.9x1.25_nomask_to_0.5x0.5_nomask_aave_da_c120522.nc'
  COMP: lnd2rof_fmapname = 'idmap'
  BASE: ocn2atm_fmapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/gx1v6/map_gx1v6_TO_fv0.9x1.25_aave.130322.nc'
  COMP: ocn2atm_fmapname = 'idmap'
  BASE: ocn2atm_smapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/gx1v6/map_gx1v6_TO_fv0.9x1.25_aave.130322.nc'
  COMP: ocn2atm_smapname = 'idmap'
  BASE: rof2lnd_fmapname = '/sems-data-store/ACME/inputdata/lnd/clm2/mappingdata/maps/0.9x1.25/map_0.5x0.5_nomask_to_0.9x1.25_nomask_aave_da_c121019.nc'
  COMP: rof2lnd_fmapname = 'idmap'
  BASE: rof2ocn_fmapname = '/sems-data-store/ACME/inputdata/cpl/cpl6/map_r05_TO_g16_aave.120920.nc'
  COMP: rof2ocn_fmapname = 'idmap'
  BASE: rof2ocn_ice_rmapname = '/sems-data-store/ACME/inputdata/cpl/cpl6/map_r05_to_gx1v6_e1000r300_090226.nc'
  COMP: rof2ocn_ice_rmapname = 'idmap'
  BASE: rof2ocn_liq_rmapname = '/sems-data-store/ACME/inputdata/cpl/cpl6/map_r05_to_gx1v6_e1000r300_090226.nc'
  COMP: rof2ocn_liq_rmapname = 'idmap'
```

**Comment by jgfouca** [comment]
For SMS_D_Ld1.ne30_oECv3_ICG.A_WCYCL1850S_CMIP6.melvin_gnu.allactive-v1cmip6:

```
2018-05-08 12:46:38: NLCOMP
Comparison failed between '/home/jgfouca/acme/scratch/SMS_D_Ld1.ne30_oECv3_ICG.A_WCYCL1850S_CMIP6.melvin_gnu.allactive-v1cmip6.C.20180508_124325_4emhrx/CaseDocs/atm_in' with '/sems-data-store/ACME/baselines/gnu/master/SMS_D_Ld1.ne30_oECv3_ICG.A_WCYCL1850S_CMIP6.melvin_gnu.a\
llactive-v1cmip6/CaseDocs/atm_in'
  BASE: ncdata = '20171228.beta3rc13_1850.ne30_oECv3_ICG.edison.cam.i.0331-01-01-00000.nc'
  COMP: ncdata = cami_mam3_Linoz_ne30np4_L72_c160214.nc'
Comparison failed between '/home/jgfouca/acme/scratch/SMS_D_Ld1.ne30_oECv3_ICG.A_WCYCL1850S_CMIP6.melvin_gnu.allactive-v1cmip6.C.20180508_124325_4emhrx/CaseDocs/lnd_in' with '/sems-data-store/ACME/baselines/gnu/master/SMS_D_Ld1.ne30_oECv3_ICG.A_WCYCL1850S_CMIP6.melvin_gnu.a\
llactive-v1cmip6/CaseDocs/lnd_in'
  BASE: finidat = '20171228.beta3rc13_1850.ne30_oECv3_ICG.edison.clm2.r.0331-01-01-00000.nc'
  COMP: finidat = I1850CLM45.ne30_oECv3.edison.intel.36b43c9.clm2.r.0001-01-06-00000_c20171023.nc'
Comparison failed between '/home/jgfouca/acme/scratch/SMS_D_Ld1.ne30_oECv3_ICG.A_WCYCL1850S_CMIP6.melvin_gnu.allactive-v1cmip6.C.20180508_124325_4emhrx/CaseDocs/mosart_in' with '/sems-data-store/ACME/baselines/gnu/master/SMS_D_Ld1.ne30_oECv3_ICG.A_WCYCL1850S_CMIP6.melvin_gn\
u.allactive-v1cmip6/CaseDocs/mosart_in'
  missing variable: 'finidat_rtm'
```

**Comment by jgfouca** [comment]
For SMS_R_Ld5.T42_T42.FSCM5A97:

```
2018-05-08 12:45:22: NLCOMP
Comparison failed between '/home/jgfouca/acme/scratch/SMS_R_Ld5.T42_T42.FSCM5A97.melvin_gnu.C.20180508_124325_4emhrx/CaseDocs/ice_in' with '/sems-data-store/ACME/baselines/gnu/master/SMS_R_Ld5.T42_T42.FSCM5A97.melvin_gnu/CaseDocs/ice_in'
Differences in namelist 'grid_nml':
  BASE: grid_file = domain.camocn.64x128_USGS_070807.nc'
  COMP: grid_file = UNSET'
  BASE: kmt_file = domain.camocn.64x128_USGS_070807.nc'
  COMP: kmt_file = UNSET'
Comparison failed between '/home/jgfouca/acme/scratch/SMS_R_Ld5.T42_T42.FSCM5A97.melvin_gnu.C.20180508_124325_4emhrx/CaseDocs/drv_in' with '/sems-data-store/ACME/baselines/gnu/master/SMS_R_Ld5.T42_T42.FSCM5A97.melvin_gnu/CaseDocs/drv_in'
  BASE: rof_gnam = 'r05'
  COMP: rof_gnam = 'null'
```

**Comment by jgfouca** [comment]
For ERS.f09_g16_g.MALISIA:

```
2018-05-08 12:44:50: NLCOMP
Comparison failed between '/home/jgfouca/acme/scratch/ERS.f09_g16_g.MALISIA.melvin_gnu.C.20180508_124325_4emhrx/CaseDocs/drv_in' with '/sems-data-store/ACME/baselines/gnu/master/ERS.f09_g16_g.MALISIA.melvin_gnu/CaseDocs/drv_in'
Differences in namelist 'cime_pes':
  BASE: atm_ntasks = 48
  COMP: atm_ntasks = 8
  BASE: cpl_ntasks = 48
  COMP: cpl_ntasks = 8
  BASE: glc_ntasks = 48
  COMP: glc_ntasks = 8
  BASE: ice_ntasks = 48
  COMP: ice_ntasks = 8
  BASE: lnd_ntasks = 48
  COMP: lnd_ntasks = 8
  BASE: ocn_ntasks = 48
  COMP: ocn_ntasks = 8
  BASE: rof_ntasks = 48
  COMP: rof_ntasks = 8
  BASE: wav_ntasks = 48
  COMP: wav_ntasks = 8
Differences in namelist 'seq_infodata_inparm':
  BASE: atm_gnam = '0.9x1.25'
  COMP: atm_gnam = 'null'
  BASE: ice_gnam = 'gx1v6'
  COMP: ice_gnam = 'null'
  BASE: lnd_gnam = '0.9x1.25'
  COMP: lnd_gnam = 'null'
  BASE: ocn_gnam = 'gx1v6'
  COMP: ocn_gnam = 'null'
  BASE: rof_gnam = 'r05'
  COMP: rof_gnam = 'null'
Comparison failed between '/home/jgfouca/acme/scratch/ERS.f09_g16_g.MALISIA.melvin_gnu.C.20180508_124325_4emhrx/CaseDocs/seq_maps.rc' with '/sems-data-store/ACME/baselines/gnu/master/ERS.f09_g16_g.MALISIA.melvin_gnu/CaseDocs/seq_maps.rc'
Differences in namelist 'seq_maps.rc':
  BASE: atm2ice_fmapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/fv0.9x1.25/map_fv0.9x1.25_TO_gx1v6_aave.130322.nc'
  COMP: atm2ice_fmapname = 'idmap'
  BASE: atm2ice_smapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/fv0.9x1.25/map_fv0.9x1.25_TO_gx1v6_blin.130322.nc'
  COMP: atm2ice_smapname = 'idmap'
  BASE: atm2ice_vmapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/fv0.9x1.25/map_fv0.9x1.25_TO_gx1v6_patc.130322.nc'
  COMP: atm2ice_vmapname = 'idmap'
  BASE: atm2ocn_fmapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/fv0.9x1.25/map_fv0.9x1.25_TO_gx1v6_aave.130322.nc'
  COMP: atm2ocn_fmapname = 'idmap'
  BASE: atm2ocn_smapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/fv0.9x1.25/map_fv0.9x1.25_TO_gx1v6_blin.130322.nc'
  COMP: atm2ocn_smapname = 'idmap'
  BASE: atm2ocn_vmapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/fv0.9x1.25/map_fv0.9x1.25_TO_gx1v6_patc.130322.nc'
  COMP: atm2ocn_vmapname = 'idmap'
  BASE: glc2lnd_fmapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/mpas.gis20km/map_mpas.gis20km_TO_fv0.9x1.25_aave.150922.nc'
  COMP: glc2lnd_fmapname = 'idmap'
  BASE: glc2lnd_smapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/mpas.gis20km/map_mpas.gis20km_TO_fv0.9x1.25_aave.150922.nc'
  COMP: glc2lnd_smapname = 'idmap'
  BASE: ice2atm_fmapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/gx1v6/map_gx1v6_TO_fv0.9x1.25_aave.130322.nc'
  COMP: ice2atm_fmapname = 'idmap'
  BASE: ice2atm_smapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/gx1v6/map_gx1v6_TO_fv0.9x1.25_aave.130322.nc'
  COMP: ice2atm_smapname = 'idmap'
  BASE: lnd2glc_fmapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/fv0.9x1.25/map_fv0.9x1.25_TO_mpas.gis20km_aave.150922.nc'
  COMP: lnd2glc_fmapname = 'idmap'
  BASE: lnd2glc_smapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/fv0.9x1.25/map_fv0.9x1.25_TO_mpas.gis20km_bilin.150922.nc'
  COMP: lnd2glc_smapname = 'idmap'
  BASE: lnd2rof_fmapname = '/sems-data-store/ACME/inputdata/lnd/clm2/mappingdata/maps/0.9x1.25/map_0.9x1.25_nomask_to_0.5x0.5_nomask_aave_da_c120522.nc'
  COMP: lnd2rof_fmapname = 'idmap'
  BASE: ocn2atm_fmapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/gx1v6/map_gx1v6_TO_fv0.9x1.25_aave.130322.nc'
  COMP: ocn2atm_fmapname = 'idmap'
  BASE: ocn2atm_smapname = '/sems-data-store/ACME/inputdata/cpl/gridmaps/gx1v6/map_gx1v6_TO_fv0.9x1.25_aave.130322.nc'
  COMP: ocn2atm_smapname = 'idmap'
  BASE: rof2lnd_fmapname = '/sems-data-store/ACME/inputdata/lnd/clm2/mappingdata/maps/0.9x1.25/map_0.5x0.5_nomask_to_0.9x1.25_nomask_aave_da_c121019.nc'
  COMP: rof2lnd_fmapname = 'idmap'
  BASE: rof2ocn_fmapname = '/sems-data-store/ACME/inputdata/cpl/cpl6/map_r05_TO_g16_aave.120920.nc'
  COMP: rof2ocn_fmapname = 'idmap'
  BASE: rof2ocn_ice_rmapname = '/sems-data-store/ACME/inputdata/cpl/cpl6/map_r05_to_gx1v6_e1000r300_090226.nc'
  COMP: rof2ocn_ice_rmapname = 'idmap'
  BASE: rof2ocn_liq_rmapname = '/sems-data-store/ACME/inputdata/cpl/cpl6/map_r05_to_gx1v6_e1000r300_090226.nc'
  COMP: rof2ocn_liq_rmapname = 'idmap'
```

**Comment by jonbob** [comment]
@jgfouca - agreed, since those two compsets really are stub components plus active landice.

**Comment by jonbob** [comment]
@jgfouca - the only one that's really concerning is the A_WCYCL1850S_CMIP6. It seems like the scripts are bypassing a set of entries at the bottom of e3sm/allactive/config_compsets.xml that sets that compset up as a hybrid run?

**Comment by jgfouca** [comment]
@jonbob , ok, my most recent commit fixes that case.

**Comment by jonbob** [comment]
@jgfouca - I have a meeting or two, but will update my codebase and check again. Thanks for all your work on this

**Comment by jonbob** [comment]
@jgfouca - I can confirm that your latest commit took care of that problem. Are the run diffs now all explainable? Looks good to me

**Comment by jgfouca** [comment]
@jonbob , I'm currently doing a full run of e3sm_integration on sandiatoss3. I don't anticipate anything other than nml diffs. All but a couple tests are complete and none have a DIFF.

**Comment by jgfouca** [comment]
@jonbob , thanks for your help btw.

**Comment by jonbob** [comment]
@jgfouca - excellent. And great job -- I'll close my review

**Comment by jgfouca** [comment]
MALI was the only DIFF since it's now using way fewer processors. I'm going to merge to next.

**Comment by rljacob** [comment]
Why did the processor count for the MALI test change?

**Comment by jgfouca** [comment]
@rljacob , because V1 was reporting that case as using grids it wasn't really using.

**Pull request merged**